toolflow/vivado/arch/axi4mm/axi4mm.tcl

```diff
@@ -375,7 +368,8 @@ namespace eval arch {
   # Connect internal clock lines.
   proc arch_connect_clocks {} {
     connect_bd_net [tapasco::subsystem::get_port "design" "clk"] \
-      [get_bd_pins -of_objects [get_bd_cells] -filter "TYPE == clk && DIR == I"]
+      [get_bd_pins -of_objects [get_bd_cells] -filter "TYPE == clk && DIR == I"] \
```

Comment:
What does line 371 do?
The clock ports of the PEs are already connected by the following line

Reply:
It is required for connecting the in- and out-interconnects